### ISSUE TITLE
Add option for migrate/down to apply/rollback specific migration

### ISSUE THREAD

**Issue by jfedyczak** [amacneil/dbmate#119]
Would be nice if I could specify which migration to run. E.g.:

```
dbmate migrate 20200213212633
dbmate down 20200213212633
```

**Comment by gudata** [comment]
I think it is a must - not nice to have.

**Comment by jayarjo** [comment]
At the moment I do something like this:

```
seq `ls /migrations | wc -l` | xargs -I INDEX  dbmate --no-dump-schema --migrations-dir /migrations down
```

to migrate down completely. It works, but it's slow like hell. Anyone any ideas how to make it faster?

**Comment by amacneil** [comment]
Out of interest, why are you migrating down completely? If it's a test harness or something it would be a lot faster to drop and recreate the database.

**Comment by jayarjo** [comment]
> Out of interest, why are you migrating down completely? If it's a test harness or something it would be a lot faster to drop and recreate the database.

@amacneil  dbmate doesn't drop when there are open connections established to the database, but can migrate down/up. I would use drop I think, if I could.

**Comment by amacneil** [comment]
Can you share why/when you are migrating down completely though? Is it at the start of your tests, or between each test, or something else?

**Comment by jayarjo** [comment]
Yes, in between test suites, to restore fresh database state.

**Comment by amacneil** [comment]
Makes sense. I'm not sure whether you considered this, but in the past when running test suites I just drop and recreate the database once at the very start (before my test harness connects to the db), then run a command like `TRUNCATE tablea, tableb, tablec CASCADE` in between each test suite. It's much faster than running all the migrations for every test.

**Comment by jayarjo** [comment]
Yeah, that's what I ended up with at the moment. Also since I run db in docker container - stopping it and running anew might also be faster.
